spinal_levels/info_label.txt

```diff
@@ -1,4 +1,4 @@
-# Spinal levels labels - generated on 2016-11-28
+# Spinal levels labels - generated on 2023-02-13
 # Keyword=IndivLabels (Please DO NOT change this line)
 # ID, name, file
 0, Spinal level C1, spinal_level_01.nii.gz
```

Comment:
## Blank C1 file

> > > Is that normal that C1 is completely blank?
> >
> > They did not include any C1 files
> 
> Hum, I am a bit reluctant to include a file labeled "C1 spinal levels" but without any label inside-- this is quite confusing for users. Maybe we should just get rid of it?

Reply:
The response from Aaron Phillips was as follows:

> Q: Is there a reason that there is no C1 label?
> A: There were not any anatomical measurements for C1 (in the Mendez paper). As a result, we did not include this in our updated estimations of spinal levels.

This partially answers the question, however we still need to decide whether or not we should remove the file.

Reply:
> This partially answers the question, however we still need to decide whether or not we should remove the file.

we definitely should. There is absolutely no point in having a 'label' file that contains only zeros. It is just confusing for everyone.

Reply:
Personally, I'm in favor of removing the C1 file, as it would it help to avoid confusion such as this user on the forum: https://forum.spinalcordmri.org/t/c1-vertebral-level-labelling/1002/4?u=joshuacwnewton

However, @jcohenadad has also promised that we would add coverage of the C1 vertebral level in https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/3997. 

So, is the idea that we would remove it just for this release, then re-add it once we have proper spinal level data? :thinking: